rskj-core/src/test/java/co/rsk/peg/ErpFederationTest.java

```diff
@@ -406,39 +406,9 @@ void testEquals_differentNumberOfMembers() {
         Assertions.assertNotEquals(federation, otherFederation);
     }
 
-    @Test
-    void testEquals_differentCreationTime() {
-        Federation otherFederation = new ErpFederation(
-            federation.getMembers(),
-            Instant.now(),
-            federation.getCreationBlockNumber(),
-            federation.getBtcParams(),
-            federation.getErpPubKeys(),
-            federation.getActivationDelay(),
-            activations
-        );
-
-        Assertions.assertNotEquals(federation, otherFederation);
-    }
-
-    @Test
-    void testEquals_differentCreationBlockNumber() {
-        Federation otherFederation = new ErpFederation(
-            federation.getMembers(),
-            federation.getCreationTime(),
-            federation.getCreationBlockNumber() + 1,
-            federation.getBtcParams(),
-            federation.getErpPubKeys(),
-            federation.getActivationDelay(),
-            activations
-        );
-
-        Assertions.assertNotEquals(federation, otherFederation);
-    }
```

Comment:
Should we remove these? We could assert that with different creation time and block number but same address they are still equal

Reply:
yes they are already removed